.github/workflows/build-action.yml

```diff
@@ -0,0 +1,84 @@
+name: Build Image
+
+# Trigger the workflow on push or pull request
+on:
+  push:
+    paths:
+      - '**.bash'
+      - '**.sh'
+      - '!test.bash'
+  pull_request:
+    paths:
+      - '**.bash'
+      - '**.sh'
+      - '!test.bash'
+
+jobs:
+  build32:
+    # Using latest Ubuntu (not the GitHub Actions default) to get latest tool versions in GitHub Actions environment
+    runs-on: ubuntu-20.04
+
+    steps:
+       - name: Checkout repository
+         uses: actions/checkout@v2
+       - name: Environment variables
+         run: sudo -E bash -c set
+       - name: Install required packages
+         run: sudo apt-get install git wget curl unzip libarchive-zip-perl dos2unix xz-utils kpartx
+       - name: imagename
+         id: imagename
+         run: |
+           echo "::set-output name=image::$(basename "$(curl "https://downloads.raspberrypi.org/raspios_lite_armhf_latest" -s -L -I  -o /dev/null -w '%{url_effective}')")"
+       - name: Cache Raspberry Pi OS image
+         uses: actions/cache@v2
+         with:
+           path: '*-raspios-*.zip'
+           key: ${{ steps.imagename.outputs.image }}
+       - name: Run build.bash rpi
+         run: |
+           sudo -E ./test.bash github
```

Comment:
"test" is a bit misleading here, right? This patches current branch to build-image/openhabian.conf, right? (see also comment below)

Reply:
More or less.